packages/ui/lib/configs/guides.tsx

```diff
@@ -0,0 +1,135 @@
+import { Button, HeaderNav, Footer } from '..'
+
+const GUIDES_HEADER = {
+  extraClass: {
+    mobile: 'bg-[#fff7e9] px-6',
+  },
+  showSocialIcons: true,
+  actions: [
+    {
+      content: <Button variant="outlined-primary">Launch App</Button>,
+    },
+  ],
+  logo: {
+    url: '/',
+    domain: 'Guides',
+  },
+  menuSections: [
+    {
+      title: 'About Unlock',
+      url: '/about',
+    },
+    {
+      title: 'Devs',
+      small: true,
+      options: [
+        {
+          title: '',
+          options: [
+            {
+              title: 'Documentation',
+              url: 'https://docs.unlock-protocol.com/',
+            },
+            {
+              title: 'Roadmap',
+              url: 'https://docs.unlock-protocol.com/governance/roadmap/',
+            },
+          ],
+        },
+      ],
+    },
+  ],
+}
+const GUIDES_FOOTER = {
+  subscriptionForm: {
+    title: 'Sign up for Updates',
+    description:
+      'Receive fresh news about Unlock, including new features and opportunities to contribute',
+    onSubmit: async (email: string) => {
+      const EMAIL_SUBSCRIPTION_FORM = {
+        portalId: '19942922',
+        formGuid: '868101be-ae3e-422e-bc86-356c96939187',
+      }
+
+      const endpoint = `https://api.hsforms.com/submissions/v3/integration/submit/${EMAIL_SUBSCRIPTION_FORM.portalId}/${EMAIL_SUBSCRIPTION_FORM.formGuid}`
+      const options = {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          fields: [
+            {
+              name: 'email',
+              value: email,
+            },
+          ],
+        }),
+      }
+      await fetch(endpoint, options)
+    },
+  },
+  logo: {
+    url: '/',
+  },
+  privacyUrl: `${window.location.origin}/privacy`,
```

Comment:
We should not have a privacy link + tos specific to guides.

Reply:
as guides is under`https://unlock-protocol.com/guides/` this will redirect to `https://unlock-protocol.com/privacy`